contracts/VTokenInterfaces.sol

```diff
@@ -157,30 +157,45 @@ abstract contract VTokenInterface is VTokenStorage {
     /**
      * @notice Event emitted when interest is accrued
      */
-    event AccrueInterest(uint256 cashPrior, uint256 interestAccumulated, uint256 borrowIndex, uint256 totalBorrows);
+    event AccrueInterest(
+        uint256 indexed cashPrior,
+        uint256 indexed interestAccumulated,
+        uint256 indexed borrowIndex,
+        uint256 totalBorrows
+    );
```

Comment:
I don't see a common use case for indexing `uint256`. Is someone going to search `AccrueInterest` events where the `cashPrior` is a specific amount? Considering the extra gas cost of indexed parameters, and the high frequency of this event, I think I wouldn't index any parameter in this event.